hooks/useGovernanceAssets.ts

```diff
@@ -850,6 +854,57 @@ export default function useGovernanceAssets() {
       isVisible: canUseAuthorityInstruction,
       packageId: PackageEnum.VsrPlugin,
     },
+    // UXD PROTOCOL
+    [Instructions.UXDInitializeController]: {
+      name: 'UXD: Initialize Controller',
+      isVisible: symbol === 'UXP',
+      packageId: PackageEnum.UXDProtocol,
+    },
+    [Instructions.UXDEditController]: {
+      name: 'UXD: Edit Controller',
+      isVisible: symbol === 'UXP',
+      packageId: PackageEnum.UXDProtocol,
+    },
+    [Instructions.UXDInitializeIdentityDepository]: {
+      name: 'UXD: Initialize Identity Depository',
+      isVisible: symbol === 'UXP',
+      packageId: PackageEnum.UXDProtocol,
+    },
+    [Instructions.UXDEditIdentityDepository]: {
+      name: 'UXD: Edit Identity Depository',
+      isVisible: symbol === 'UXP',
+      packageId: PackageEnum.UXDProtocol,
+    },
+    [Instructions.UXDMintWithIdentityDepository]: {
+      name: 'UXD: Mint with Identity Depository',
+      isVisible: symbol === 'UXP',
+      packageId: PackageEnum.UXDProtocol,
+    },
+    [Instructions.UXDRedeemWithIdentityDepository]: {
+      name: 'UXD: Redeem with Identity Depository',
+      isVisible: symbol === 'UXP',
+      packageId: PackageEnum.UXDProtocol,
+    },
+    [Instructions.UXDRegisterDepository]: {
+      name: 'UXD: Register a Depository',
+      isVisible: symbol === 'UXP',
+      packageId: PackageEnum.UXDProtocol,
+    },
+    [Instructions.UXDEditDepository]: {
+      name: 'UXD: Edit a Depository',
+      isVisible: symbol === 'UXP',
+      packageId: PackageEnum.UXDProtocol,
+    },
+    [Instructions.UXDMint]: {
+      name: 'UXD: Mint',
+      isVisible: canUseAnyInstruction,
+      packageId: PackageEnum.UXDProtocol,
+    },
+    [Instructions.UXDRedeem]: {
+      name: 'UXD: Redeem',
+      isVisible: canUseAnyInstruction,
+      packageId: PackageEnum.UXDProtocol,
+    },
```

Comment:
nit, no need to prefix with `UXD`: 

```suggestion
    [Instructions.UXDInitializeController]: {
      name: 'Initialize Controller',
      isVisible: symbol === 'UXP',
      packageId: PackageEnum.UXDProtocol,
    },
    [Instructions.UXDEditController]: {
      name: 'Edit Controller',
      isVisible: symbol === 'UXP',
      packageId: PackageEnum.UXDProtocol,
    },
    [Instructions.UXDInitializeIdentityDepository]: {
      name: 'Initialize Identity Depository',
      isVisible: symbol === 'UXP',
      packageId: PackageEnum.UXDProtocol,
    },
    [Instructions.UXDEditIdentityDepository]: {
      name: 'Edit Identity Depository',
      isVisible: symbol === 'UXP',
      packageId: PackageEnum.UXDProtocol,
    },
    [Instructions.UXDMintWithIdentityDepository]: {
      name: 'Mint with Identity Depository',
      isVisible: symbol === 'UXP',
      packageId: PackageEnum.UXDProtocol,
    },
    [Instructions.UXDRedeemWithIdentityDepository]: {
      name: 'Redeem with Identity Depository',
      isVisible: symbol === 'UXP',
      packageId: PackageEnum.UXDProtocol,
    },
    [Instructions.UXDRegisterDepository]: {
      name: 'Register a Depository',
      isVisible: symbol === 'UXP',
      packageId: PackageEnum.UXDProtocol,
    },
    [Instructions.UXDEditDepository]: {
      name: 'Edit a Depository',
      isVisible: symbol === 'UXP',
      packageId: PackageEnum.UXDProtocol,
    },
    [Instructions.UXDMint]: {
      name: 'Mint',
      isVisible: canUseAnyInstruction,
      packageId: PackageEnum.UXDProtocol,
    },
    [Instructions.UXDRedeem]: {
      name: 'Redeem',
      isVisible: canUseAnyInstruction,
      packageId: PackageEnum.UXDProtocol,
    },
```

Reply:
but if you don't use the package filter, you will not really know what it corresponds to

Reply:
fixed